plugins/thunderbolt/fu-plugin-thunderbolt.c

```diff
@@ -84,6 +84,29 @@ fu_plugin_init (FuPlugin *plugin)
 	fu_plugin_add_firmware_gtype (plugin, NULL, FU_TYPE_THUNDERBOLT_FIRMWARE_UPDATE);
 }
 
+// gboolean
+// fu_plugin_backend_device_added (FuPlugin *plugin, FuDevice *device, GError **error)
+// {
+// 	g_autofree gchar *denied_vidpid = fu_plugin_get_config_value (plugin,
+// 								"DeniedDevice");
+
+// 	if (denied_vidpid != NULL) {
+// 		g_autofree gchar *udev_vidpid = g_strdup_printf ("%04x%04x",
+// 			    fu_udev_device_get_vendor (FU_UDEV_DEVICE (device)),
+// 			    fu_udev_device_get_model (FU_UDEV_DEVICE (device)));
+
+// 		if (g_strcmp0 (denied_vidpid, udev_vidpid) == 0) {
+// 			g_debug ("ignored thunderbolt device 0x%s", udev_vidpid);
+// 			g_set_error (error, FWUPD_ERROR,
+// 				     FWUPD_ERROR_NOT_SUPPORTED,
+// 				     "ignored thunderbolt device 0x%s",
+// 				     udev_vidpid);
+// 			return FALSE;
+// 		}
+// 	}
+// 	return TRUE;
+// }
+
```

Comment:
don't add this stuff

Reply:
thanks for the catch, missed to remove this test.